pages/auth_page.py

```diff
@@ -0,0 +1,60 @@
+from locators import LocatorsSignIn
+
+
+class AuthPage():
+    """Sign in page with login method for any type of person"""
+    def __init__(self, base_obj):
+        self.browser = base_obj.browser
+        self.header = base_obj.header
+        self.locators = LocatorsSignIn
+
+    def clear_boxes(self):
+        self.browser.clear_element(self.locators.EMAIL_SIGN_IN)
+        self.browser.clear_element(self.locators.PASSWORD_SIGN_IN)
+
+    def enter_sign_in_email(self, email):
+        self.browser.send_keys(self.locators.EMAIL_SIGN_IN, email)
+
+    def enter_sign_in_password(self, password):
+        self.browser.send_keys(self.locators.PASSWORD_SIGN_IN, password)
+
+    def enter_credentials(self, username, password):
+        self.enter_sign_in_email(username)
+        self.enter_sign_in_password(password)
+
+    def click_sign_in(self):
+        self.browser.click_element(self.locators.BUTTON_SIGN_IN)
+
+    def login_user(self, username, password):
+        self.header.click_icon()
+        self.header.click_log_in()
+        self.enter_sign_in_email(username)
```

Comment:
why not `enter_credentials` method?

Reply:
or in this case, we don't need `enter_credentials` method, because we don't use it at all

Reply:
I use this method couz it's easier to see on which step test failed. 
enter_credentials use Dima in his tests